src/views/viewer/ChannelView/ChannelView.tsx

```diff
@@ -271,6 +272,12 @@ export const ChannelView: React.FC = () => {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, [])
```

Comment:
Just as a sidenote - this is not related to the issue, but I think we could remove this comment and add instead: 
```tsx
  const initialRender = useRef(true)
  // At mount set the tab from the search params
  useEffect(() => {
    if (initialRender.current) {
      const tabIndex = TABS.findIndex((t) => t === currentTabName)
      if (tabIndex === -1) setSearchParams({ 'tab': 'Videos' }, { replace: true })
      initialRender.current = false
    }
  })
```

Reply:
This way `useEffect` will be fired on every render

Reply:
I don't think so, keep in mind that I used here `useRef`. It should run only on mount. This is very similar to https://stackoverflow.com/a/53180013

Reply:
Yeah, you are right. I didn't look closely, I only saw that there was no array ;)